src/metabase/core.clj

```diff
@@ -111,9 +111,6 @@
   ;; Bootstrap the event system
   (events/initialize-events!)
   (init-status/set-progress! 0.7)
-  ;; Now initialize the task runner
-  (task/init-scheduler!)
-  (init-status/set-progress! 0.8)
```

Comment:
@camsaul I wanted to call attention to this in particular. Initialization is now performed by the call to `(task/start-scheduler!)` below.

Reply:
To expand on this, it is to prevent errors in tasks that connect to an existing sample database, which will still be H2 v1 until the sample data setup runs. I didn't see any obvious reason not to combine the initialization with startup, and tests pass locally. CI could prove me wrong 😬